code/modules/reagents/chemistry/reagents/food_reagents.dm

```diff
@@ -246,6 +246,16 @@
 	description = "A salt made of sodium chloride. Commonly used to season food."
 	reagent_state = SOLID
 	color = "#FFFFFF" // rgb: 255,255,255
+	overdose_threshold = 30
+	
+/datum/reagent/consumable/sodiumchloride/overdose_start(mob/living/M)
+	M << "<span class='userdanger'>You're too salty!</span>"
+
+
+/datum/reagent/consumable/sodiumchloride/overdose_process(mob/living/M)
+	M.adjustToxLoss(1.5*REM, 0)
+	M.adjustFireLoss(1*REM, 0)
```

Comment:
@PKPenguin321 You know that chems don't react all instantly right?
You'd have 40 ticks to crit which is ~100 seconds to get yourself some burn healing or something to get to medbay at least.